charts/refarch-template/charts/backend/templates/tests/test-connection.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: "{{ .Chart.Name . }}-test-connection"
+  labels:
+    app: {{ .Chart.Name }}
+  annotations:
+    "helm.sh/hook": test
+spec:
+  containers:
+    - name: wget
+      image: busybox
```

Comment:
die frage ist, ob man das intern so machen soll, weil man ja nicht direkt von Docker pullen soll

Reply:
@hupling M.E. passt das so. Die Tests laufen ja nur in Github.

Reply:
also bei Sonarqube bin ich mir ziemlich sicher, dass mit einem `helm install` auch der Testpod ausgerollt wird. Das zeigt, dass das Helm-Chart richtig installiert wurde

Reply:
Deine Frage zielt also auf unsere internen Templates ab? Dann bitte ein Issue dazu öffnen und diese conversation schließen.

Reply:
dann lassen wir es so wie es ist.